docs/proposals/pepxxx_wheel_variant_support.md

```diff
@@ -183,6 +765,103 @@ Several alternative approaches were considered and ultimately rejected:
 
 ![pytorch selector](../assets/images/pytorch_variant_selector.webp)
 
+### Wheel Variants
+
+#### Wheel filename
+
+- Adding the variant label as a third component (before the build tag) with additional `~` characters. This approach
+  was ultimately rejected, as adding it as a last component made it easier to distinguish different variants,
+  and achieved the same goals.
+
+- Using labels in addition to the variant hash. This approach was rejected because it caused unnecessary increase
+  of filename length. As the specification evolved and made it unnecessary to include the hash in the filename,
+  it was suggested to replace it with a human-readable label instead.
+
+- Automatically generating human-readable labels by providers. This idea did not fit well with very limited variant
```

Comment:
I wouldn't reject this outright, for the cases where you have e.g. only a cpu provider or only a GPU provider that's totally feasible.

Reply:
Perhaps, but I don't think this would be trivial to design. For example, looking at the previous example of four possible variants: the `cu122` and `x8664v4` variants are both single-property, so would fall into using the custom label. However, `cu122 + x8664v4` would no longer get a custom label, which would be quite inconsistent.